### PR TITLE
feat: 增加支付宝小程序兼容和打包处理

### PR DESCRIPTION
**Pull request by RGCHN**
## Summary by CodeRabbit

- **New Features**
  - Added support for the Alipay platform across multiple packages.
  - Exported new functions and classes in the Orientation Transformer package.

- **Enhancements**
  - Improved import organization for better code readability and maintenance.
  - Updated JSON variable assignment to use dynamic URLs in the demo package.

- **Bug Fixes**
  - Removed unnecessary console log statements for cleaner output.
  - Adjusted conditional checks to include mini-program environment detection.

- **Documentation**
  - Added comments for better understanding of specific code functionalities.

- **Refactor**
  - Reordered and grouped imports for better organization without altering functionality.

### CODE REVIEW

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: plugin-packages/orientation-transformer/src/utils/device.ts [hunk 1]
```diff
@@ -4,3 +4,7 @@ export function isIOS () {
 
   return ver ? parseInt(ver[2], 10) : 0;
 }
+
+export function isMiniProgram () {
+  return 'miniProgram' in navigator;
```
[the remark this answers is not in the document]
Comment (RGCHN) [reply plugin-packages/orientation-transformer/src/utils/device.ts:9]: 这个判断是支付宝和微信小程序都用的 支付宝是 另外之后降级插件也需要通过navigator里面字段判断 所以加了个miniProgram字段

[the remark this answers is not in the document]
Comment (yiiqii) [reply plugin-packages/orientation-transformer/src/utils/device.ts:9]: `isAlipayMiniApp() || isWechatMiniApp()`
那只有判断吧，支付宝的小程序的判断还是以官方方式安全点

File: packages/effects-webgl/src/gl-program.ts [hunk 2]
```diff
@@ -84,7 +84,8 @@ export class GLProgram implements Disposable {
       geometry.vaos[programId] = vao;
     }
 
-    if (vao) {
+    // 兼容小程序下不支持vao
+    if (vao && vao.vao) {
```
Comment (yiiqii) [inline packages/effects-webgl/src/gl-program.ts:88]: 这个我没看懂。。

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-webgl/src/gl-program.ts (hunk 2 above)
Comment (RGCHN) [reply packages/effects-webgl/src/gl-program.ts:88]: vao是创建的vao对象, vao.vao 在 webgl1 下是vao扩展，因为小程序不支持，所以要判断下这个扩展能否使用，不能的话就走后面重新绑定 buffer 的逻辑

**Review by yiiqii — APPROVED** [review]

**Pull request merged**